android/app/build.gradle

```diff
@@ -105,6 +105,16 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+    signingConfigs {
+        release {
+            if (project.hasProperty('MYAPP_RELEASE_STORE_FILE')) {
+                storeFile file(MYAPP_RELEASE_STORE_FILE)
+                storePassword MYAPP_RELEASE_STORE_PASSWORD
```

Comment:
@housseindjirdeh that's the point, or something I do not understand. This is necessary to create its own release version (because for example it is not always enough for Android simulator, you need to check on a real device).